src/config/app.config.ts

```diff
@@ -0,0 +1,64 @@
+import { InternalServerErrorException } from '@nestjs/common';
+import { LogLevel } from '@nestjs/common/services/logger.service';
+import { registerAs } from '@nestjs/config';
+import * as Joi from 'joi';
+import * as path from 'path';
```

Comment:
to all native NodeJS module add 'node' like this from 'node:path', and move it to the top